static/js/types/common.d.ts

```diff
@@ -0,0 +1,27 @@
+export interface LxdConfigPair {
```

Comment:
Common is a place that tends to grow because it is not specific. What do you think about splitting this into `config` and `device` files?

Reply:
Sure, done @ [bebfcc3](https://github.com/canonical/lxd-ui/commit/bebfcc3ad5861579eb99377d4e5e112ec510c0fd)